src/components/CreateComment.jsx

```diff
@@ -1,75 +1,131 @@
+import { useEffect, useState, forwardRef } from "react";
+import { toast } from "react-hot-toast";
 import { FaArrowCircleRight } from "react-icons/fa";
-import { postComment } from "services";
-import { useDispatch } from "react-redux";
+import { postComment, editComment } from "services";
+import { useSelector, useDispatch } from "react-redux";
 import { ProfileImage } from "./ProfileImage";
-import { useState } from "react";
-export function CreateComment({ postId }) {
-  const [formData, setFormData] = useState({
+import { Spinner } from "./Spinner";
+export const CreateComment = forwardRef(({ postId, commentToEdit }, ref) => {
```

Comment:
clever use of forwardRef
nicely done @kjais1720

Reply:
Thank you 🤗